src/components/Accordion/Accordion.tsx

```diff
@@ -89,7 +89,7 @@ class Accordion extends AutoControlledComponent<Extendable<IAccordionProps>, any
     /** Accessibility behavior if overridden by the user. */
     accessibility: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
 
-    /** Custom styles to be applied for component. */
+    /** Styles to apply to this component instance **/
```

Comment:
seems that we might consider a more explicit way to express the same thing, like
`CSS styles to apply to ... `

This will make this comment more descriptive, as well as will go along the content provided in other comments in terms of specificity (`className` as an example)